<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QIQCapDlg</class>
 <widget class="QWidget" name="QIQCapDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,0,0,0,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Trigger Source</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>comboBoxTrigSrc</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxTrigSrc">
           <item>
            <property name="text">
             <string>Free Run</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Restart Marker</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marker 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marker 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marker 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marker 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>User Define Marker</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ext Trigger</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ext Trigger A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ext Trigger B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ext Trigger C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ext Trigger D</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IF Power</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Trigger Edge</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>comboBoxTrigEdge</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxTrigEdge">
           <item>
            <property name="text">
             <string>Posedge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Negedge</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Trigger Offset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>lineEditTrigOffset</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTrigOffset"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Trigger Gap</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>lineEditTrigGap</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTrigGap"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Capture Offset</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>lineEditCapOffset</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditCapOffset"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Timeout(us)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>lineEditTimeout</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTimeout"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Samples</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>lineEditSamples</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSamples"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Threshold</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>lineEditThreshold</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditThreshold"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrameSlotTrigWidget" name="widgetFrameSlotTrig" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLineEdit" name="lineEditPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSetPath">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QPushButton" name="pushButtonFrameSlotTrigStart">
         <property name="text">
          <string>Frame Slot Trigger Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFrameSlotTrigStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="pushButtonCap">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAbort">
         <property name="text">
          <string>Abort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSetAll">
         <property name="text">
          <string>Set All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,0,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,0">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonShowTD">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>TD</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkBoxShowI">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>I</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxShowQ">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Q</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>19</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,0">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonShowDFT">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>DFT</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBoxMaxHold">
             <property name="text">
              <string>Max Hold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxMinHold">
             <property name="text">
              <string>Min Hold</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>State</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>lineEditPwrMesState</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPwrMesState"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Power</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>lineEditPwrMesPwr</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPwrMesPwr"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Peak</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>lineEditPwrMesPeak</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPwrMesPeak"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPwrMes">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Power Meas</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QwtPlot" name="plot"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFrameSlotTrigWidget</class>
   <extends>QWidget</extends>
   <header>qframeslottrigwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
